packages/parcel-transformer/src/index.ts

```diff
@@ -108,17 +108,21 @@ export default new Transformer<ParcelTransformerOpts>({
       plugins: config.transformerBabelPlugins ?? undefined,
     });
 
-    return {
```

Comment:
What happens when skipping over this? Do things still get transformed / compiled / extracted?

Reply:
This shouldn't happen in practice, because the AST will generate if there's some code. This is just making the behaviour more explicit, where the result will be ignored if there's no AST.

Reply:
Alright. Does Parcel need some types updated now since program can be undefined?

Reply:
probably! These values are `any` :/